features/step_definitions/operators.rb

```diff
@@ -160,3 +160,35 @@
   cb_name = 'operator_channel_name' unless cb_name
   cb[cb_name] = cluster_version('version').channel.split('-')[1]
 end
+
+Given /^I wait(?: up to (\d+) seconds)? for operator "([^"]*)" to check conditions: "([^"]*)"$/ do | time, operator, conditions |
```

Comment:
`I wait ... operator "..." to check conditions: "Conditionname=Value [...]"`
I think `check conditions:` is hard to understand what it means. Let's use `become`, as many step definitions used.

Reply:
what about below?
```
I wait ... operator "..." to become ready with conditions: "Conditionname=Value [...]"
```

Reply:
Many step definitions used `ready` too. `ready` means normal. So don't use `ready`.
`I wait ... operator "..." to become "Conditionname=Value [...]"` is simple and straightforward.

Reply:
This looks like better.